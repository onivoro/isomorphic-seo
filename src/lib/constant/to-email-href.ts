export const toEmailHref = (email: string, subject: string, prefix= '') => email && `mailto:${email}?subject=${encodeURIComponent(prefix + subject)}`;
