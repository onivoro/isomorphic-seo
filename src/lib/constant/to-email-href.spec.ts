import { toEmailHref } from './to-email-href';
const testEmail = 'e@ma.il';
const testSubject = 'subjetooo';
const testPrefix = 'RE:RE:REEE: ';

describe('toEmailHref', () => {
    describe('success case', () => {
        it('worx', () => {
            expect(toEmailHref(testEmail, testSubject, testPrefix))
                .toMatchSnapshot();
        });

    });
});
