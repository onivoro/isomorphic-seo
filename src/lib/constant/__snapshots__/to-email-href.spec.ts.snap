// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toEmailHref success case worx 1`] = `"mailto:e@ma.il?subject=RE%3ARE%3AREEE%3A%20subjetooo"`;
